com.github.joseexposito.touche.yml

```diff
@@ -0,0 +1,38 @@
+app-id: com.github.joseexposito.touche
+runtime: org.gnome.Platform
+runtime-version: '3.38'
+sdk: org.gnome.Sdk
+command: com.github.joseexposito.touche
+finish-args:
+  - '--share=ipc'
+  - '--socket=fallback-x11'
+  - '--socket=wayland'
+  - '--filesystem=home'
+  - '--filesystem=host:ro'
+modules:
+  - name: libwnck
+    buildsystem: meson
+    config-opts:
+      - '-Dinstall_tools=false'
+      - '-Dgtk_doc=false'
+    sources:
+      - type: git
+        url: https://gitlab.gnome.org/GNOME/libwnck.git
```

Comment:
I would use a tarball here from download.gnome.org if possible.

Reply:
Thanks for the tip, I just updated it.